src/libs/ComponentSpecs.js

```diff
@@ -40,7 +40,6 @@ const IS_DEBUG_MODE = (function isDebugMode() {
 }());
 const DEFAULT_SCAN_TAGS = ['a', 'button'];
 let pageInitViewportDetectionTimeout = null;
-let pageInitViewportDetected = false;
```

Comment:
this causing scope pollution when two different files changing the same global value